Integrations/integration-RSANetWitness_v11.1.yml

```diff
@@ -433,10 +433,7 @@ script:
         """
         LOG('Requestig to update incident ' + incident_id)
 
-        body = {
-            'assignee': assignee,
-            'status': status
-        }
+        body = "{{\"status\": \"{}\",\"asignee\": \"{}\"}}".format(status, assignee)
```

Comment:
Just making sure you indeed meant for double curly braces?

Reply:
Yeah, the double curly are required because of the `format` function, otherwise it raises a `KeyError` exception.